libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -232,6 +261,20 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     this.#_radioType = value ?? 'info';
   }
 
+  /**
+   * The text to display as the radio button's label. Use this instead of the `sky-radio-label` when the label is text-only.
+   * @preview
+   */
+  @Input()
+  public labelText: string | undefined;
```

Comment:
include documentation that this will be the `aria-labelledby` text as well

Reply:
Other way around. If `labelText` exists, `aria-labelledby` isn't set.

Reply:
ah sorry i got that confused with `aria-label`. those already have the deprecated warning pointing to use labelText so this comment is resolved anyways